staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -483,6 +484,17 @@ func (r *Reflector) watch(ctx context.Context, w watch.Interface, resyncerrc cha
 	var err error
 	retry := NewRetryWithDeadline(r.MaxInternalErrorRetryDuration, time.Minute, apierrors.IsInternalError, r.clock)
 
+	defer func() {
```

Comment:
We are just concern about missing stacktrace for recovered panics passed to `panicCh`. This is not needed.